lib/github-package.js

```diff
@@ -71,6 +72,7 @@ export default class GithubPackage {
           this.setActiveContext(WorkdirContext.absent());
         }
       }),
+      ContextMenuInterceptor,
```

Comment:
I'm not 100% sure I understand why this is necessary, if registrations only happen on mount and are removed on unmount.